volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -820,6 +823,26 @@ def d_ancestor(self, ancestor_dentry):
             current_dentry = current_dentry.d_parent
         return None
 
+    def get_subdirs(self) -> interfaces.objects.ObjectInterface:
```

Comment:
I feel as though the addition of this method should require a bump to the PATCH number at least, if not the MINOR number.  The only question is whether it's the whole framework or if this is part of the LinuxUtilities which is already versioned separately...

Reply:
This is in the `dentry` extension object, but I don't think we've ever changed any versions when there were changes in this area. Hmm, it is not related to `LinuxUtilities`.

Reply:
Ok, I think I see what's going on here (and the LinuxUtilities checks I asked for earlier).  We've got bits of the core framework (extensions) that aren't versioned other than under the umbrella of the main core, that are using LinuxUtilities which we gave its own version number to.  So what *should* be/have been happening is the main core bumps (at least PATCH but probably minor) whenever a linux extension was added, only I'm not sure we always did that because people hate screwing with version numbers...  5:S  Probably we can fix this by going up by making sure all the extensions we want to add are in place, and then going to (2,9,0) and all plugins that need/make use of it require that as a minimum framework version.  Hopefully that'll clear everything up and even if it's a little broken in the middle, it'll give people a new version number they can then rely on to have the functionality they want/need.